lib/oli/utils/error_logger.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Oli.Utils.ErrorLogger do
+  require Logger
+
+  def log_error(e, label \\ "Unexpected error") do
+    Logger.error(label <> ": " <> Kernel.inspect(e))
```

Comment:
There is already a `log_error` function in Oli.Utils. Should we move that function into this new module?